OpenRA.Mods.Common/Traits/Upgrades/DeployToUpgrade.cs

```diff
@@ -53,6 +53,9 @@ public class DeployToUpgradeInfo : ITraitInfo, Requires<UpgradeManagerInfo>
 		[Desc("Sound to play when undeploying.")]
 		public readonly string UndeploySound = null;
 
+		[Desc("Is the initial deployment of this actor permanent?")]
+		public readonly bool IsDeploymentPermanent = false;
```

Comment:
`Permanent` would be sufficient as a name. `Deployment` is redundant, and inconsistent with our usual `Deploy` terminology.